interface/forms/procedure_order/common.php

```diff
@@ -1338,17 +1422,6 @@ class="col-form-label col-md-2"><?php echo xlt('Status'); ?></label>
                                     'list_id' => 'ord_status'), $row['order_status'] ?? '');
                                 ?>
                             </div>
-                            <label for="form_billing_type"
```

Comment:
Did AI remove this or is billing no longer needed either?

Reply:
No I moved these to make room for needed specimen fields.